perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -198,8 +221,29 @@ def PrepareVMEnvironment(self):
       if self.is_static:
         self.SnapshotPackages()
       self.SetupPackageManager()
+    self.ConfigureVMKernel()
     self.BurnCpu()
 
+  def ConfigureVMKernel(self):
+    """Apply {procfs,sysfs}_config to VM."""
+
+    path_components = None
+
+    def ConfigureSubtree(subtree):
+      for name, value in subtree.iteritems():
+        path_components.append(name)
+        if isinstance(value, basestring):
+          self.RemoteCommand('echo "%s" | sudo tee %s' %
```

Comment:
Do we have a way to mock the RemoteCommand so that we can get back what will be run?  That might be too much testing ... but it shouldn't be that brittle.